lib/types/number.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
-
 var MIN_INTEGER = -100000000,
     MAX_INTEGER = 100000000,
     DEFAULT_PRECISION   = .00001;
```

Comment:
how about adding 0 in front of .00001? I know it's valid JS syntax, but 0.00001 is hack-less and looks cleaner for me.